### PR TITLE
feat: Implement Facebook stories share

### PR DESCRIPTION
**Pull request by Quynh-Nguyen**
# Overview
Implement Facebook Stories share.
Follow this document of Facebook: https://developers.facebook.com/docs/sharing/sharing-to-stories/ios-developers

# Test Plan
- Add more handler for share to Facebook.
- Tested with my React Native project on iPhone 7.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -526,10 +527,49 @@ Supported options for INSTAGRAM_STORIES:
...
 
+### Static Values for Facebook Stories
+These can be assessed using Share.Social property
+For eg.
+```javascript
+import Share from 'react-native-share';
+
+const shareOptions = {
+    method: Share.FacebookStories.SHARE_BACKGROUND_AND_STICKER_IMAGE,
+    backgroundImage: 'http://urlto.png',
```
Comment (MateusAndrade) [inline README.md:538]: Does this accept a `base64`?

File: README.md [hunk 2]
```diff
@@ -526,10 +527,49 @@ Supported options for INSTAGRAM_STORIES:
...
+    stickerImage: 'data:image/png;base64,<imageInBase64>', //or you can use "data:" link
```
Comment (MateusAndrade) [inline README.md:539]: Do this works sharing with a URL?

File: README.md [hunk 3]
```diff
@@ -526,10 +527,49 @@ Supported options for INSTAGRAM_STORIES:
...
+    backgroundBottomColor: '#fefefe',
+    backgroundTopColor: '#906df4',
+    attributionURL: 'http://deep-link-to-app', //in beta
+    appId: '219376304', //facebook appId
```
Comment (MateusAndrade) [inline README.md:543]: This is the appId on Facebook Pages?

File: index.js [hunk 4]
```diff
@@ -92,6 +92,7 @@ type Options = {
   failOnCancel?: boolean,
   showAppsToView?: boolean,
   saveToFiles?: boolean,
+  appId?: string,
```
Comment (MateusAndrade) [inline index.js:95]:
```suggestion
  appId: string,
```

This should be required, no?

File: README.md (hunk 1 above)
Comment (Quynh-Nguyen) [reply README.md:538]: Yes it's working both with base64 data image and url.

File: README.md (hunk 2 above)
Comment (Quynh-Nguyen) [reply README.md:539]: Yes it's working with both base64 data image and url.

File: README.md (hunk 3 above)
Comment (Quynh-Nguyen) [reply README.md:543]: Yes that's right, it's required from Facebook.
With Instagram we doesn't need, but with Facebook they required.

File: README.md (hunk 1 above)
Comment (MateusAndrade) [reply README.md:538]: Maybe we can let implicit  that this param accept both `url` and `base64`.

File: README.md (hunk 2 above)
Comment (MateusAndrade) [reply README.md:539]: Maybe we can let implicit  that this param accept both `url` and `base64`.

**Comment by Quynh-Nguyen** [comment]
> Also, could you provide an example to test this? Since we're using Semantic Release, whenever this gets merged a new version will be published, so we need to be careful. 😅
> 
> I will try testing this here and after that, we can merge this. 🎉

Yah this is the testing result.

image

File: README.md (hunk 1 above)
Comment (Quynh-Nguyen) [reply README.md:538]: > Maybe we can let implicit that this param accept both `url` and `base64`.
Yes. So should me update the example source code on README file?

File: README.md (hunk 2 above)
Comment (Quynh-Nguyen) [reply README.md:539]: > Maybe we can let implicit that this param accept both `url` and `base64`.

Yes. So should me update the example source code on README file?

File: README.md (hunk 1 above)
Comment (MateusAndrade) [reply README.md:538]: Yeah, sure! I approved this before seeing your comment. 😅

File: README.md (hunk 2 above)
Comment (MateusAndrade) [reply README.md:539]: Same here.

File: README.md (hunk 1 above)
Comment (MateusAndrade) [inline README.md:538]:
```suggestion
    backgroundImage: 'http://urlto.png', // url or an base64 string
```

File: README.md (hunk 2 above)
Comment (MateusAndrade) [inline README.md:539]:
```suggestion
    stickerImage: 'data:image/png;base64,<imageInBase64>', //or you can use "data:" url
```

**Pull request merged**